src/components/SingleArticle.jsx

```diff
@@ -33,19 +36,49 @@ export default function SingleArticle(){
      if (loading) { return <div>Loading...</div>; } 
      if (error) { return <div>Error: {error.message}</div>; }
 
-    return( <div key ={article.article_id}> 
-    <div id= "singleArticle">
-    <p className = "articleItem"><strong>Title: </strong> {article.title}</p><div className= "container"><img src= {article.article_img_url} id= "articleImage"/></div>
-    <p className = "articleItem"><strong>Author: </strong> {article.author}</p>
-    <p className = "articleItem"><strong>Topic: </strong> {article.topic}</p>
+     function handleClick(event) {
+        event.preventDefault();
+        if (!showComments) {
+           
+            getCommentsById(article_id).then((fetchedComments) => {
+                setComments(fetchedComments || []);
+                setShowComments(true);
+            });
+        } else {
+            setShowComments(false);
+        }
+    }
+    return (
+        <div key={article.article_id}>
+            <div id="singleArticle">
+                <p className="articleItem"><strong>Title: </strong> {article.title}</p>
+                <p className="articleItem"><strong>Author: </strong> {article.author}</p>
+                <p className="articleItem"><strong>Topic: </strong> {article.topic}</p>
+                <p className="articleItem"><strong>Article: </strong> {article.body}</p>
+                <img src={article.article_img_url} alt="Article image" className="articleImg" />
+                <p className="articleItem"><strong>Votes: </strong> {article.votes}</p>
+                <p className="articleItem"><small>Posted on </small> {formatDate(dateString)}</p>
+                <p><strong>Comments:</strong> {article.comment_count}</p>
+                <button className="button" onClick={handleClick}>{showComments ? "Hide Comments" : "View Comments"}</button>
     
-    <p className = "articleItem"><strong>Date Posted:</strong> {formatDate(dateString)}</p>
-    <div id = "voteButton">
-    <p className = "articleItem"><strong>Votes:</strong> {article.votes}</p><button className = "button"  > Vote! </button>
-    </div>
-    <p><strong>Article:</strong>  {article.body}</p>
-    <p><strong>Comments:</strong>  {article.comment_count}</p>
-    </div>
-    </div>
-    )
-}
\ No newline at end of file
+                {showComments && (
+                    <div id="commentsSection">
+                        <div className="commentsContainer">
+                            {comments.length > 0 ? (
+                                comments.map((comment) => (
+                                    <div key={comment.comment_id} className="comment">
+                                        <p><strong>{comment.author}</strong> says:</p>
```

Comment:
Is the use of strong to show any specific importance or is it just to make it bold? If the latter just use `b` or `css` to achieve this